folio-security/src/main/java/org/folio/security/integration/keycloak/utils/ClientBuildUtils.java

```diff
@@ -36,7 +39,8 @@ public static Keycloak buildKeycloakAdminClient(String clientSecret, KeycloakPro
   }
 
   private static ResteasyClient buildResteasyClient(TlsProperties tls) {
-    var clientBuilder = newBuilder().hostnameVerifier(INSTANCE);
+    var clientBuilder = newBuilder().hostnameVerifier(
+      IS_HOSTNAME_VERIFICATION_DISABLED ? NoopHostnameVerifier.INSTANCE : DEFAULT_HOSTNAME_VERIFIER);
```

Comment:
Can't we pass this as a method parameter, and have it properly configurable? Using system properties for this seems very crude and non-agile.

Reply:
This is how it works in the JDK for JDK HttpClient so I simply "copy-pasted" the solution because HttpClient implementation is in the internal package and is not available for unnamed modules.
This way, we have a single unified system property that defines the behavior for all HttpClient implementations we use for mgr-* and mod-*-keycloak modules